### DISCUSSION TITLE
posts/exploring-website-development-for-personal-sites/

### DISCUSSION THREAD

**Discussion by giscus** [nerdymomocat/nerdymomocat.github.io#8] [Announcements]
# posts/exploring-website-development-for-personal-sites/

Tools for easy personal websites

https://nerdymomocat.github.io/posts/exploring-website-development-for-personal-sites/

**Comment by joodaloop** [comment]
Check out https://getkirby.com/

**Reply by nerdymomocat** [reply]
Kirby does look cool! I was considering more all-in-one options, but this is a CMS I had not heard of before
